src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java

```diff
@@ -175,4 +181,17 @@ private boolean isInAirspace(@NotNull Craft craft) {
         corners.add(new MovecraftLocation(hitbox.getMaxX(), hitbox.getMaxY(), hitbox.getMaxZ()));
         return corners;
     }
+
+    private boolean canManOverboard(Player player, Craft craft) {
+        if(craft.getDisabled())
+            return false;
+        if (craft.getW() != player.getWorld())
+            return false;
+
+        double telX = (craft.getHitBox().getMinX() + craft.getHitBox().getMaxX())/2D;
+        double telZ = (craft.getHitBox().getMinZ() + craft.getHitBox().getMaxZ())/2D;
+        double telY = craft.getHitBox().getMaxY() + 1;
+        Location telPoint = new Location(craft.getW(), telX, telY, telZ);
```

Comment:
To simplify this we could merely use craft.getHitBox.getMidpoint, which is an approximation but should be sufficient and much simpler.